src/main/java/com/toofifty/easyblastfurnace/state/InventoryState.java

```diff
@@ -44,26 +44,28 @@ public void update()
 
     public int getChange(int itemId)
     {
-        return getQuantity(itemId) - getPreviousQuantity(itemId);
+        return getQuantity(new int[]{itemId}) - getPreviousQuantity(itemId);
     }
 
     public boolean hasChanged(int itemId)
     {
         return getChange(itemId) != 0;
     }
 
-    public int getQuantity(int itemId)
+    public int getQuantity(int[] itemIds)
     {
         load();
+        int total = 0;
 
-        return inventory.count(itemId);
-    }
+        for (int itemId : itemIds) {
+            total += inventory.count(itemId);
+        }
 
-    public boolean has(int itemId)
-    {
-        return getQuantity(itemId) > 0;
+        return total;
     }
 
+    public boolean has(int[] itemIds) { return getQuantity(itemIds) > 0; }
```

Comment:
If you use [variable arguments](https://www.geeksforgeeks.org/variable-arguments-varargs-in-java/), you can achieve the same functionality without having to add the array initialisers everywhere

```suggestion
    public boolean has(int ...itemIds) { return getQuantity(...itemIds) > 0; }
```

It can then be used like so

```java
inventory.has(ItemID.ICE_GLOVES);
inventory.has(ItemID.SMITHING_CAPE, ItemID.SMITHING_CAPET);

Reply:
:O